changelogs/unreleased/remove-redundant-attribute-exception.yml

```diff
@@ -0,0 +1,3 @@
+description: Remove redundant try/except case
+change-type: patch
+destination-branches: [master, iso5]
```

Comment:
https://github.com/inmanta/inmanta-core/commit/f9246f24b55a7045222b8be8c7c271af31e02b08 was merged on iso4, so this change has to be merged on iso4 as well.